MaxDuino.ino

```diff
@@ -1353,8 +1352,6 @@ void changeDir() {
     tmpdir.close();
   }
   getMaxFile();
-  currentFile=0;
```

Comment:
I removed these because these are now set in getMaxFile (which avoids some duplication)